dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/AbstractInterfaceConfigTest.java

```diff
@@ -118,6 +118,8 @@ public void checkApplication2() throws Exception {
     public void testLoadRegistries() throws Exception {
         System.setProperty("dubbo.registry.address", "addr1");
         InterfaceConfig interfaceConfig = new InterfaceConfig();
+        // FIXME: now we need to check first, then load
+        interfaceConfig.checkRegistry();
```

Comment:
Maybe we should give `loadRegistries` another name that can better reflect what it does. It is misleading since it loads nothing except for converts RegistryConfigs to URLs.